knife-supermarket.gemspec

```diff
@@ -14,5 +14,5 @@ Gem::Specification.new do |s|
   s.homepage    = 'https://github.com/cwebberOps/knife-supermarket'
   s.license       = 'Apache 2.0'
   s.require_paths = ["lib"]
-  s.add_dependency "chef", ">= 0.10.10"
+  s.add_dependency "chef", ">= 0.10.10", "<= 12.11.18"
```

Comment:
@lamont-granquist not sure how to do that; was thinking about
```
if Gem::Version.new(Chef::VERSION) > Gem::Version.new("12.11.18")
  puts <<EOF
This gem has been deprecated and the `knife supermarket` subcommands have been moved in to core Chef. Please ensure you have Chef 12.12 or newer, and then uninstall this gem.
EOF
else
old code here
```
but that doesn't output the log…